README.md

```diff
@@ -1,3 +1,66 @@
 # apostrophe-seo
 
-Test.
+SEO for [ApostropheCMS](http://apostrophecms.org/).
+
+Add useful meta fields to `apostrophe-custom-pages`, `apostrophe-pieces`, and `apostrophe-global`.
```

Comment:
This is the inside baseball of how it works. Say what it's good for. "Adds useful meta fields for SEO to all pages and pieces."